_internals/system-views.md

```diff
@@ -4,47 +4,47 @@ title: Babelfish system view
 nav_order: 4
 ---
 
-## Babelfish system views 
+## Babelfish system views
 
-Once in a while, an application has to access an MS SQL-compatible system view to
+Once in a while, an application has to access an MS SQL-compatible system view
+or a server side function to
 work properly. This is especially true for graphical user interfaces. However,
-accessing an MS SQL-compatible system view is also necessary for some applications 
```

Comment:
remove the line "This is especially true for graphical user interfaces" -- this is nonsense

Reply:
Does a GUI like SSMS have to access database metadata? Yes.
Is that done in some other way?
I just want to double-check.